pkgs/tools/compression/flips/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, gtk3, pkg-config }:
+
+stdenv.mkDerivation {
+  pname = "flips";
+  version = "1.31";
```

Comment:
I only briefly looked at the site but I'm wondering where this version comes from.

Reply:
It comes from https://github.com/Alcaro/Flips/commit/24e6c77eb8d1ed6d0eb6da400c67ea229eca2278 (the initial git revision when the code was moved to VCS), not sure why it wasn't tagged.

Reply:
Sounds good. Maybe you can ping upstream to tag the next release of you don't mind.